build-resources.gradle

```diff
@@ -3,12 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-//preferably try to main the alphabetical order
+//preferably try to maintain the alphabetical order
 ext.versionMap = [
         junitJupiter : '5.7.2',
         mockito : '3.11.2',
         opentelemetryProto : '1.0.1-alpha',
         opensearchVersion : '1.1.0'
 ]
 
-ext.coreProjects = [project(':data-prepper-api'), project(':data-prepper-core'), project('data-prepper-plugins:common')]
\ No newline at end of file
+ext.coreProjects = [project(':data-prepper-api'), project(':data-prepper-core'), project('data-prepper-plugins:common')]
+ext.mavenArtifactProjects = [project(':data-prepper-api'), project('data-prepper-plugins:common')]
```

Comment:
Do we need to publish the common plugins? How does this help plugin developers?

Reply:
If we don't need them, can we add the publishing instructions to the plugin build.gradle file instead?

Reply:
I took another look at `common`. I believe we can remove it from the publication.

> If we don't need them, can we add the publishing instructions to the plugin build.gradle file instead?
Can you elaborate on this request?

Reply:
Sorry, I mixed plugins and api in my original ask. I was thinking we could have the publishing instructions closer to the package that we are publishing but this works as is.